src/test/groovy/edu/ie3/datamodel/io/connectors/SqlIndividualTimeSeriesMetaInformationTest.groovy

```diff
@@ -0,0 +1,51 @@
+package edu.ie3.datamodel.io.connectors
+
+import edu.ie3.datamodel.io.naming.timeseries.ColumnScheme
+import edu.ie3.datamodel.io.naming.timeseries.IndividualTimeSeriesMetaInformation
+import spock.lang.Shared
+import spock.lang.Specification
+
+class SqlIndividualTimeSeriesMetaInformationTest extends Specification {
+
+    @Shared
+    private final UUID randomUUID = UUID.fromString("80d1bbec-79df-4ae2-b60a-34a2c3f77408")
+
+    @Shared
+    private final ColumnScheme columnScheme = ColumnScheme.ACTIVE_POWER
+
+    @Shared
+    private final String tableName = "its_p_80d1bbec-79df-4ae2-b60a-34a2c3f77408"
+
+    def "SqlIndividualTimeSeriesMetaInformation is properly created with standard constructor"() {
+        when:
+        def sqlMetaInf = new SqlConnector.SqlIndividualTimeSeriesMetaInformation(randomUUID, columnScheme, tableName)
+
+        then:
+        sqlMetaInf.uuid == randomUUID
+        sqlMetaInf.columnScheme == columnScheme
+        sqlMetaInf.tableName == tableName
+    }
+
+    def "SqlIndividualTimeSeriesMetaInformation is properly created with secondary constructor"() {
+        given:
+        def metaInf = new IndividualTimeSeriesMetaInformation(randomUUID, columnScheme)
+
+        when:
+        def sqlMetaInf = new SqlConnector.SqlIndividualTimeSeriesMetaInformation(metaInf, tableName)
+
+        then:
+        sqlMetaInf.uuid == randomUUID
+        sqlMetaInf.columnScheme == columnScheme
+        sqlMetaInf.tableName == tableName
+    }
+
+    def "SqlIndividualTimeSeriesMetaInformation returns a proper String representation"() {
```

Comment:
Well, I guess we don't need such kind of a test, right?

Reply:
Without it, test coverage is too low